csm/core/services/rgw/s3/users.py

```diff
@@ -102,3 +102,23 @@ async def remove_key(self, **remove_key_body):
         uid = remove_key_body.get(const.UID)
         Log.debug(f"Removing key for S3 IAM user by uid = {uid}")
         return await self.execute_request(const.REMOVE_KEY_OPERATION, **remove_key_body)
+
+    @Log.trace_method(Log.DEBUG)
+    async def add_user_caps(self, **request_body):
+        """
+        This method will add user caps to S3 IAM user.
+        :param **request_body: User body kwargs
+        """
+        uid = request_body.get(const.UID)
+        Log.debug(f"Add User caps S3 IAM user = {uid}")
```

Comment:
Make this Log.Info and change "Add User caps S3 IAM user" -> "Adding user capabilities for {uid}"

Reply:
updated